grafana-plugin/src/pages/index.tsx

```diff
@@ -144,13 +144,6 @@ export const pages: { [id: string]: PageDefinition } = [
     path: getPath('cloud'),
     action: UserActions.OtherSettingsWrite,
   },
-  {
-    icon: 'gf-logs',
-    id: 'organization-logs',
```

Comment:
hmm, are you sure that nobody uses Org Logs page by direct link? I've got 404 on `/api/plugin-proxy/grafana-oncall-app/api/internal/v1/organization_logs/` request but maybe it's just hidden by feature flag on backend side or something....